tests/core_weekly/test_date_util.py

```diff
@@ -8,15 +8,20 @@ class TestDateUtil(unittest.TestCase):
     def setUp(self):
         self.date_util = DateUtil()
 
+    def test_get_date_range_from_week_1_2020(self):
+        self.assertEqual(self.date_util.get_date_range_from_week(1, 2020), '2019-12-30..2020-01-05')
+
     def test_get_date_range_from_week_23_2020(self):
         self.assertEqual(self.date_util.get_date_range_from_week(23, 2020), '2020-06-01..2020-06-07')
 
     def test_get_date_range_from_week_38_2020(self):
         self.assertEqual(self.date_util.get_date_range_from_week(38, 2020), '2020-09-14..2020-09-20')
 
-    def test_get_date_range_from_week_38(self):
```

Comment:
This test cannot be relied on because depending on the year, it sometimes is true, sometimes is wrong. A test that rely on "now" is not a good test I think. or needs to be updated every year